docsite/source/rules.html.md

```diff
@@ -191,7 +191,7 @@ class DistanceContract < Dry::Validation::Contract
   end
 
   rule(:kilometers, :miles) do
-    if key?(:kilometers) ^ key?(:miles)
+    if key?(:kilometers) && key?(:miles)
```

Comment:
`unless key?(:kilometers) ^ key?(:miles)` would make more sense IMO :)

Reply:
I did consider whether that might be the intention, but then the message wouldn't make sense.
This class looks like it actually requires 2 rules -

- "must only contain one of: kilometers, miles"
- "must contain one of: kilometers, miles"